src/hooks/use-claim-mento.tsx

```diff
@@ -37,6 +39,26 @@ export const useClaimMento = ({
   }
 
   const addresses = mento.addresses[chainId];
+  const preparedClaimArgs = prepareArgs({
+    allocation,
+    address,
+    merkleProof,
+    proof,
+    validUntil,
+    approvedAt,
+    fractalId,
+  });
+  const gasEstimate = useEstimateGas(
+    preparedClaimArgs
+      ? {
+          address: addresses.Airgrab as Address,
+          abi: Airdrop,
+          functionName: "claim",
+          args: preparedClaimArgs,
+          account: address as Address,
+        }
+      : null,
```

Comment:
You could use the `query` part of this hook to `enable` the query

This hook I think auto updates while there's something present so in theory, the intended pattern is to provide an params and select whether it runs